assets/src/edit-story/components/library/karma/libraryTabs.karma.js

```diff
@@ -85,5 +90,23 @@ describe('LibraryTabs integration', () => {
       expect(getExpandedPanes()).toEqual([mediaPane]);
       await fixture.waitOnScreen(mediaPane);
     });
+
+    it('should return focus to current tab when pressing mod+alt+1', async () => {
+      const { textTab } = fixture.editor.library;
+
+      // Click tab
+      await fixture.events.mouse.clickOn(textTab);
+      await waitFor(() => fixture.editor.library.text);
+      expect(textTab).toHaveFocus();
+
+      // Click elsewhere
+      await fixture.events.click(fixture.editor.canvas.header.title);
+      expect(textTab).not.toHaveFocus();
+
+      // Return focus with shortcut
+      await fixture.events.keyboard.shortcut('mod+alt+1');
+      expect(textTab).toHaveFocus();
+      await fixture.snapshot('text tab has focus');
```

Comment:
The snapshot doesn't actually contain the focus outline, even though it shows up when running karma tests locally. Not sure why.